onboarding/fastapi.md

```diff
@@ -0,0 +1,61 @@
+This is a quick starter guide for FastAPI applications.
+
+You can check further instructions for FastAPI integration in our <a href="https://docs.rollbar.com/docs/python" target="_blank" rel="noopener">Complete Python Documentation</a>.
+FastAPI integration examples are also available in our <a href="https://github.com/rollbar/pyrollbar/tree/master/rollbar/examples/fastapi" target="_blank" rel="noopener">GitHub repository</a>.
+
+## Installation
+
+Use the <a href="https://github.com/rollbar/pyrollbar" target="_blank" rel="noopener">pyrollbar</a> SDK for sending events to Rollbar from your FastAPI application.
+
+Install pyrollbar with pip:
+
+```shell script
+pip install rollbar
+```
+
+## Test your Integration
+
+The example below is a recommended way to integrate Rollbar with FastAPI.
+This example requires FastAPI version 0.41.0 or later. There are other possible integrations (e.g. via middleware) that are compatible with older FastAPI versions.
+More information can be found in our <a href="https://docs.rollbar.com/docs/python" target="_blank" rel="noopener">documentation</a>.
```

Comment:
@powellguaglione-rollbar @natesweet 
I'm not convinced about this note. Should we add alternative instructions for older FastAPI versions? This also duplicates the header.

Reply:
Is there a way to merge both lines in order to avoid repetitions with first (header) message?

`The example below is a recommended way to integrate Rollbar with FastAPI 0.41.0 or later. For alternative methods supporting older FastAPI versions, please read our &lt;a href="https://docs.rollbar.com/docs/python" target="_blank" rel="noopener"&gt;Python documentation&lt;/a&gt;.`

Reply:
Other option is having 1 example for FastAPI 0.41.0 or later and 1 for older FastAPI versions:
## Test FastAPI 0.41.0 or later
...
## Test older FastAPI versions
...

Reply:
Thanks for the feedback. I updated the paragraph.